Minimal Web API/DocumentEditorHelper.cs

```diff
@@ -17,12 +17,15 @@
 
 namespace DocumentEditorCore
 {
-    public class DocumentEditor
+    /// <summary>
+    /// 
```

Comment:
Contains the server-side helper functions for JavaScript Document editor control.